core/src/test/kotlin/org/opensearch/alerting/core/model/ScheduleTest.kt

```diff
@@ -301,6 +301,16 @@ class ScheduleTest : XContentTestBase {
         assertTrue(intervalSchedule.runningOnTime(null))
     }
 
+    @Test
+    fun `execution time matches across different time zones`() {
+        val pdtClockCronSchedule = CronSchedule("* * * * *", ZoneId.systemDefault())
+        val utcClockCronSchedule = CronSchedule("* * * * *", ZoneId.systemDefault())
+        val instant = Instant.ofEpochSecond(1539615226L)
+        val pdtNextExecution = pdtClockCronSchedule.getExpectedNextExecutionTime(instant.atZone(ZoneId.of("America/Los_Angeles")).toInstant(), null)
+        val utcNextExecution = utcClockCronSchedule.getExpectedNextExecutionTime(instant.atZone(ZoneId.of("UTC")).toInstant(), null)
```

Comment:
Same comments as from this [PR](https://github.com/opensearch-project/job-scheduler/pull/189/files#diff-3f7f018bb67e7a9174045925acee2bf79dcf5c0db01e730e73c42881b10321beR45-R46).

Reply:
Good point. Refactored the test.